.pre-commit-config.yaml

```diff
@@ -26,6 +26,5 @@ repos:
     -   id: mypy
         additional_dependencies: [types-requests]
 
-
 default_language_version:
-    python: python3.8
+    python: python3
```

Comment:
what version is this?

Reply:
I copied over from [`ape-solidity`](https://github.com/ApeWorX/ape-solidity/blob/main/.pre-commit-config.yaml) pre-commit yaml, so this might be wrong